src/routers/dev/index.ts

```diff
@@ -0,0 +1,34 @@
+import type { AuthenticatedRequest } from '../../types/network';
+
+import express, { Response, NextFunction } from 'express';
+import auth from '../../middleware/auth';
+import positions from '../../data/positions';
+import PositionModel from '../../models/position';
+
+const router = express.Router();
+
+// create positions if they don't exist
+router.post(
+  '/positions',
+  auth,
+  async (req: AuthenticatedRequest, res: Response, next: NextFunction) => {
+    try {
+      positions.forEach(async (p) => {
+        const existingPosition = await PositionModel.find({ title: p.title });
+        if (!existingPosition) {
+          const position = new PositionModel({
+            ...p,
+            isOpen: true,
+            occupant: req.user?._id,
+          });
+          await position.save();
+        }
+      });
+      res.send();
+    } catch (err) {
+      next(err);
```

Comment:
Btw how do feel about [this `express-async-errors` package](https://github.com/davidbanham/express-async-errors). It would let us not have to catch just to send to next.

Reply:
I'm down for using it. If you want you can include it the next time that you're creating a route and we can ticket the refactor for all routes if it looks good.

Reply:
sg!